olm-catalog/serverless-operator/serverless-operator.v1.2.0.clusterserviceversion.yaml

```diff
@@ -84,6 +84,12 @@ spec:
         displayName: Version
         path: version
       version: v1alpha1
+    required:
+      - description: A list of namespaces in Service Mesh
+        displayName: Istio Service Mesh Member Roll
+        kind: ServiceMeshMemberRoll
```

Comment:
We should also depend on ServiceMeshControlPlane here. It won't make any difference in effect, but we really depend on both of these CRDs.

Reply:
Sure.